vmss_images/02-capture-images.yml

```diff
@@ -0,0 +1,30 @@
+# Description
+# ===========
+# This playbook creates an Azure VM with public IP, and then dumps it using facts
+
+- name: Create Azure VM
```

Comment:
update name and description above to align with playbook

Reply:
yes, I still have a few issues like this to fix :-)